core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -255,6 +257,9 @@ public void start(WorkerNetAddress address) throws IOException {
               (int) ServerConfiguration.getMs(PropertyKey.WORKER_BLOCK_HEARTBEAT_INTERVAL_MS),
                   ServerConfiguration.global(), ServerUserState.global()));
     }
+
+    // Mounts the embedded Fuse application if configured
+    mFuseManager.start();
```

Comment:
shall this be protected by a flag or config?

Reply:
Add a new configuration `WORKER_FUSE_ENABLED` to decide whether to start fuse manager, thanks!